### PR TITLE
Update minor things.

### PR DESCRIPTION
**Pull request by drvinceknight**
- Fix percentage not appearing
- Tweak the abstract
- Consistant IPD
- Include some more references.

### CODE REVIEW

**Comment by drvinceknight** [comment]
Also now includes some changes made by @Nikoleta-v3 :+1:

File: main.tex [hunk 1]
```diff
@@ -44,18 +44,19 @@
...
 
 \section{Introduction}\label{sec:introduction}
 
-The iterated prisoner's dilemma (IPD) is a common model in game theory,
+The Iterated Prisoner's Dilemma (IPD) is a common model in game theory,
 frequently used  to understand the evolution of cooperative behaviour from complex
 dynamics \cite{Axelrod1984}.
 
-In this manuscript we use the Axelrod library \cite{axelrodproject, knight2016open},
+In this manuscript the Axelrod library \cite{knight2016open, axelrodproject},
```
Comment (marcharper) [inline main.tex]: How about "this manuscript uses the Axelrod library... principal goal."?

File: main.tex [hunk 2]
```diff
@@ -259,7 +262,7 @@ \subsection{Finite State Machines}\label{sec:fsm}
 \begin{figure}[!hbtp]
     \centering
     \includestandalone[height=.3\textheight]{./assets/fsm}
-    \caption{Diagrammatic representation of the Finite state machine Archetype}
+    \caption{Diagrammatic representation of the Finite state machine Archetype.}
```
Comment (marcharper) [inline main.tex]: Let's be consistent with capitalization: Finite State Machine like the other captions

File: main.tex [hunk 3]
```diff
@@ -287,12 +290,12 @@ \subsection{Finite State Machines}\label{sec:fsm}
         noisy tournament}
         \label{fig:fsm16noise}
     \end{subfigure}%
-    \caption{Trained sixteen state finite state machine players}
+    \caption{Trained sixteen state finite state machine players.}
```
Comment (marcharper) [inline main.tex]: Capitals?

**Review by marcharper — APPROVED** [review]
Looks good, just a couple of very minor suggestions

File: main.tex (hunk 1 above)
Comment (drvinceknight) [reply main.tex]: Much better :+1:

File: main.tex (hunk 3 above)
Comment (drvinceknight) [reply main.tex]: Yup and yup :+1:

File: main.tex [hunk 4]
```diff
@@ -259,7 +262,11 @@ \subsection{Finite State Machines}\label{sec:fsm}
 \begin{figure}[!hbtp]
     \centering
     \includestandalone[height=.3\textheight]{./assets/fsm}
-    \caption{Diagrammatic representation of the Finite state machine Archetype}
+<<<<<<< HEAD
```
Comment (marcharper) [inline main.tex]: Merge conflict yikes!

File: main.tex [hunk 5]
```diff
@@ -287,12 +294,16 @@ \subsection{Finite State Machines}\label{sec:fsm}
         noisy tournament}
         \label{fig:fsm16noise}
     \end{subfigure}%
-    \caption{Trained sixteen state finite state machine players}
+<<<<<<< HEAD
```
Comment (marcharper) [inline main.tex]: same

File: main.tex (hunk 4 above)
Comment (drvinceknight) [reply main.tex]: Yuck! How did I miss that. Fixing now :) :+1:

**Pull request merged**